packages/amplify-graphql-transformer-interfaces/src/graphql-api-provider.ts

```diff
@@ -56,7 +56,19 @@ export interface OpenIDConnectConfig {
 }
 
 export interface LambdaConfig {
+  /**
+   * Function name, excluding optional `-{env}` suffix.
+   */
   lambdaFunction: string;
+
+  /**
+   * Actually lambda Arn, if provided this will circumvent the arn construction when building the API auth mode config.
```

Comment:
```suggestion
   * The ARN of an existing Lambda function. If provided, this will circumvent the ARN construction when building the API auth mode config. The ARN must refer to the same function
   * named in `lambdaFunction`.
```